src/app/api/BookAPI/Book/route.ts

```diff
@@ -3,253 +3,251 @@ import { prisma } from "@/db";
 import { NextResponse } from "next/server";
 
 export async function POST(request: Request) {
-	const req = await request.json();
-	let { barcode, isbn, title, author, genre } = req;
-
-	if (typeof barcode == "number") {
-		barcode = barcode.toString();
-	}
-
-	try {
-		const combinedBookData = await prisma.scannedBook.create({
-			data: {
-				book: {
-					connectOrCreate: {
-						where: {
-							barcode: barcode,
-						},
-						create: {
-							barcode: barcode,
-							isbn: isbn,
-							title: title,
-							author: author,
-							genre: genre,
-						},
-					},
-				},
-			},
-		});
-
-		return NextResponse.json(combinedBookData);
-	} catch (error: any) {
-		console.log(error);
-		return NextResponse.json(
-			{ error: "Failed Adding Local Book..." },
-			{ status: 500 }
-		);
-	}
+  const req = await request.json();
+  let { barcode, isbn, title, author, genre } = req;
+
+  if (typeof barcode == "number") {
+    barcode = barcode.toString();
+  }
+
+  try {
+    // TODO this is not setting it to park but should, must be some stored shit going on
+    const combinedBookData = await prisma.scannedBook.create({
+      data: {
+        book: {
+          connectOrCreate: {
+            where: {
+              barcode: barcode,
+            },
+            create: {
+              barcode: barcode,
+              isbn: isbn,
+              title: title,
+              author: author,
+              genre: genre,
+              park: true,
```

Comment:
Can ignore trying to add the park to true here, as it will be an @default(true) in the schema, so can be left out in this section

Believe error could be to do with req args being passed, do not believe genre is passed at the moment?